src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -190,6 +191,15 @@ class EmojiPickerMenu extends Component {
         document.addEventListener('mousemove', this.mouseMoveHandler);
     }
 
+    /**
+     * @param {*} data
+     * @param {Number} index
+     * @returns {Object}
+     */
+    getItemLayout(data, index) {
+        return {length: CONST.EMOJI_PICKER_ITEM_HEIGHT, offset: CONST.EMOJI_PICKER_ITEM_HEIGHT * index, index};
+    }
+
```

Comment:
Can you quickly add comments for the params and a brief comments for the method also?

Reply:
Added